catroid/src/main/java/org/catrobat/catroid/common/Constants.java

```diff
@@ -299,6 +299,8 @@
 
 	public static final int MAX_NUMBER_OF_CHECKED_TAGS = 3;
 
+	public static final int MILLISECONDS_IN_A_SECOND = 1000;
```

Comment:
Since this value is only used within PauseForBeatsAction, please move it there as a private compile-time constant. https://kotlinlang.org/docs/properties.html#compile-time-constants